packages/core/src/app/address/AddressForm.spec.tsx

```diff
@@ -29,107 +29,90 @@ describe('AddressForm Component', () => {
 
     it('renders all fields based on formFields', () => {
         component = mount(
-            <LocaleContext.Provider value={ localeContext }>
-                <Formik
-                    initialValues={ {} }
-                    onSubmit={ noop }
-                >
-                    <AddressForm
-                        fieldName="address"
-                        formFields={ formFields }
-                    />
+            <LocaleContext.Provider value={localeContext}>
+                <Formik initialValues={{}} onSubmit={noop}>
+                    <AddressForm fieldName="address" formFields={formFields} />
                 </Formik>
-            </LocaleContext.Provider>
+            </LocaleContext.Provider>,
         );
 
-        expect(component.find('[name="address.shouldSaveAddress"]').exists())
-            .toEqual(false);
+        expect(component.find('[name="address.shouldSaveAddress"]').exists()).toBe(false);
 
-        expect(component.find(DynamicFormField).length).toEqual(formFields.length);
+        expect(component.find(DynamicFormField)).toHaveLength(formFields.length);
     });
 
     it('renders DynamicFormField with expected props', () => {
         component = mount(
-            <LocaleContext.Provider value={ localeContext }>
-                <Formik
-                    initialValues={ {} }
-                    onSubmit={ noop }
-                >
+            <LocaleContext.Provider value={localeContext}>
+                <Formik initialValues={{}} onSubmit={noop}>
                     <AddressForm
                         fieldName="address"
-                        formFields={ formFields }
-                        shouldShowSaveAddress={ true }
+                        formFields={formFields}
+                        shouldShowSaveAddress={true}
```

Comment:
Just noticed, `eslint` does not force space usages inside a `{}`.
Both `{formFields}` and `{ formFields }` can pass now.